caddyhttp/proxy/proxy_test.go

```diff
@@ -1037,7 +1037,7 @@ func TestBasicAuth(t *testing.T) {
 	}{
 		{"Nil Both", nil, nil},
 		{"Nil Upstream User", nil, url.UserPassword("username", "password")},
-		{"Nil Client User", url.UserPassword("usename", "password"), nil},
+		{"Nil Client User", url.UserPassword("username", "password"), nil},
```

Comment:
This might be intentional, if so, there are better ways to express this.

Reply:
As long as the tests pass this is fine.